Base/Common.xcconfig

```diff
@@ -172,3 +172,6 @@ RUN_CLANG_STATIC_ANALYZER = YES
 
 // Don't treat unknown warnings as errors, and disable GCC compatibility warnings and unused static const variable warnings
 WARNING_CFLAGS = -Wno-error=unknown-warning-option -Wno-gcc-compat -Wno-unused-const-variable -Wno-nullability-completeness
+
+// Enable no common blocks build setting
```

Comment:
Could you explain why we're adding this (the warning) and add a brief description of what the setting does?